README.md

```diff
@@ -155,6 +160,12 @@ Mobile Wallet acts as a **self-custodial** Identity Wallet that:
 
 This implementation involves 3 distinct participants that have different responsibilities:
 
+### PFI
+
+The PFI operates a KCC issuer, as well as a tbDEX exchange.
```

Comment:
do we feel like making explicit mention to `tbDEX` and `PFI`s is necessary vs. just `Issuer`? 

i know we mention of KCC issuance being an _extension_ of the work being done for tbDEX here:

![image](https://github.com/TBD54566975/known-customer-credential/assets/4887440/af3af945-7243-4d9a-a129-b7eb9676c33f)


thinking using "Issuer" would make it more generalized. wdyt @tomdaffurn ?

Reply:
Was unsure about explicit links to tbDEX. I added one for `presentation_definition_id` cos I think a reader will not know where that comes from. Same for this paragraph - goal is to explain how a wallet might find the PD ID

Reply:
@tomdaffurn    I think we may be able to lean on on the fact that PDs in and of themselves are an established concept especially given that they're referenced in [presentation exchange](https://identity.foundation/presentation-exchange/spec/v2.0.0/#presentation-definition) [credential manifest](https://identity.foundation/credential-manifest/#credential-manifest) and [OID4VP](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-presentation_definition-par). ID is a required property of PDs. 

We can always mention how this works with tbDEX as an _example_. Vs. first classing tbdex and PFI as core concepts.